oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/CoreModuleConfig.java

```diff
@@ -217,6 +217,20 @@ public class CoreModuleConfig extends ModuleConfig {
      */
     private int uiMenuRefreshInterval = 20;
 
+    /**
+     * The service cache refresh interval, default 10s
+     */
+    @Setter
+    @Getter
+    private int serviceCacheRefreshInterval = 10;
+
+    /**
+     * The max size of service cache, default 10000
+     */
+    @Setter
+    @Getter
+    private int serviceCacheMaxSize = 10_000;
```

Comment:
```suggestion
    private int serviceCacheMaxSize = 100_000;
```

As it is only a name cache,  a little more larger should be fine.

Reply:
@kezhenxu94 As scrolling is there, this size should be fine, right?